aws-s3-transfer-manager/src/operation/download/handle.rs

```diff
@@ -23,11 +23,11 @@ pub struct DownloadHandle {
     /// Object metadata.
     pub(crate) object_meta: OnceCell<ObjectMetadata>,
 
-    /// The object content
+    /// The object content and metadata
```

Comment:
trivial: "and metadata" makes it sound like there's 1 metadata, and this is where you get it ... which is confusing because there's also the `object_meta` field.

Probably more interesting to point out that the object data will come in chunks. You can point out that each chunk has its own metadata...
```suggestion
    /// The object content, in chunks, and the metadata for each chunk
```
or not mention that detail, since most people probably aren't interested
```suggestion
    /// The object content, in chunks
```